src/test/scala/net/ruippeixotog/scalascraper/Examples.scala

```diff
@@ -17,6 +18,14 @@ object ExampleMatchers {
   val errs = validatorsAt[Int]("error-matchers")
 }
 
+object ProxyApp extends App {
+  ProxyUtils.setProxy("localhost", 3128)
+  val browser = Browser()
+  val doc = browser.get("http://observador.pt")
+
+  println("=== OBSERVADOR HTTP & HTTPS PROXY ===")
```

Comment:
In this case I think there's not to much to add. In the last commit I added the removeProxy and a comment with the exception that you should get if you're under a proxy.